assembl/static2/js/app/client.js

```diff
@@ -27,6 +29,31 @@ const myFragmentMatcher = new IntrospectionFragmentMatcher({
 
 const dataIdFromObject = o => o.id;
 
+const networkInterface = createNetworkInterface({
+  uri: getFullPath('graphql', { slug: getDiscussionSlug() }),
+  opts: {
+    credentials: 'same-origin'
+  }
+});
+
+// trace every graphql operation in sentry
+networkInterface.use([
```

Comment:
Hmm, not choice. This is a choice and perhaps should be discussed. Because the backend is logging all of this in the assembl.log file, line by line, as JSON, as a middleware too. But, there isn't a web-reader available for it. Are we going to log in both places? Also, what about the spamminess of these logs on Sentry. We've usually used Sentry to log for warn level and up.

Reply:
If we want clear Graphql errors, we need this here because the error is reported by the js sentry client and not the python one (well in fact, it depends because for mutations decorated with `abort_transaction_on_exception` it is done on the python side).

Without this, we are blind about what cause the error (we don't know the query, we can find the component but it does not help a lot).

My guess is that the best would be to abort transactions also for queries and report the error in the sentry python sdk. We have a big work to do on reporting as the whole. But until then, I think we should merge this to be able to debug errors in the meanwhile.

Reply:
Yeah, I thought about it some more, and I agree with you. Keep this in. We haven't made any decision (ie, putting the stacktrace on sentry for the abort_on_transaction). So, until we do a reporting user story, this stays.